packages/node_modules/pouchdb-mapreduce/src/index.js

```diff
@@ -356,8 +356,8 @@ function getDocsToPersist(docId, view, docIdsToChangesAndEmits) {
   var metaDocId = '_local/doc_' + docId;
   var defaultMetaDoc = {_id: metaDocId, keys: []};
   var docData = docIdsToChangesAndEmits.get(docId);
-  var indexableKeysToKeyValues = docData.indexableKeysToKeyValues;
-  var changes = docData.changes;
+  var indexableKeysToKeyValues = docData[0];
+  var changes = docData[1];
```

Comment:
using an array here is more efficient than `{changes: ..., indexableKeysToKeyValues: ...}`. the keys don't need to be spelled out explicitly because it's all internal code